frontend/.env

```diff
@@ -0,0 +1,4 @@
+VUE_APP_MAPBOX_TOKEN=mapbox_token
```

Comment:
As everyone does not have the same configuration, we should name this file `.env-sample` and add `.env` in the `.gitignore`